base/redactor.go

```diff
@@ -35,6 +40,15 @@ func redact(args []interface{}) []interface{} {
 	return args
 }
 
+func (redactorFuncSlice RedactorFuncSlice) Redact() string {
```

Comment:
also this, as `[]RedactorFunc` would satisfy `[]Redactor`

Reply:
Not sure what you mean here?

Reply:
I meant that now redactFunc implements the reactor interface, you're allowed to build a slice of redactFuncs without a new type and you can remove all this code too